gsl/span

```diff
@@ -347,6 +347,7 @@ class span
 public:
     // constants and types
     using element_type = ElementType;
+    using value_type = ElementType;    
```

Comment:
This should be `using value_type = std::remove_cv_t<ElementType>;` - in standard library parlance, value types are never cv-qualified.